File: spec/php/HelloWorldTest.php
<?php
namespace Kaitai\Struct\Tests;

class HelloWorldTest extends TestCase {
    public function testHelloWorld() {
        $r = HelloWorld::fromFile(self::SRC_DIR_PATH . '/fixed_struct.bin');
        // Access property through direct call
        $this->assertEquals(0x50, $r->one());
    }
}

File: spec/go/hello_world_test.go
package spec

import (
	"os"
	"testing"

	"github.com/stretchr/testify/assert"
	"github.com/kaitai-io/kaitai_struct_go_runtime/kaitai"

	. "test_formats"
)

func TestHelloWorld(t *testing.T) {
	f, err := os.Open("../../src/fixed_struct.bin")
	if err != nil {
		t.Fatal(err)
	}
	s := kaitai.NewStream(f)

	var h HelloWorld
	err = h.Read(s, &h, &h)
	if err != nil {
		t.Fatal(err)
	}

	assert.Equal(t, byte(0x50), h.One)
}
